src/App.tsx

```diff
@@ -1,156 +1,82 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  ChangeEvent,
+  useEffect,
+  useState,
+} from 'react';
+
+import { getUsers } from './api/users';
+import { getTodos } from './api/todos';
+import { Login } from './Login';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Filter } from './components/Filter/Filter';
+import { NewTodo } from './components/NewTodo/NewTodo';
+import NotificationError from
+  './components/NotificationError/NotificationError';
 
-const USER_ID = 0;
+const USER_ID = localStorage.getItem('userId');
 
 export const App: React.FC = () => {
+  const [task, setTask] = useState('');
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+
+  const [error, setError] = useState<string>('');
```

Comment:
Redundant spaces here
```suggestion
  const [task, setTask] = useState('');
  const [todos, setTodos] = useState<Todo[]>([]);
  const [error, setError] = useState<string>('');
```